pyproject.toml

```diff
@@ -23,13 +23,13 @@ dependencies = [
 
 [project.optional-dependencies]
 cpu = [
-    'llm-foundry[all-cpu]@git+https://github.com/mosaicml/llm-foundry.git@main#egg=llmfoundry',
+    'llm-foundry[all-cpu]@git+https://github.com/mosaicml/llm-foundry.git@ricky-fsdp2-temp-version#egg=llmfoundry',
```

Comment:
Using this since it has the dependency on the composer PR with the FSDP changes, will remove this after that PR has been merged in.